sklearn/linear_model/ransac.py

```diff
@@ -68,8 +68,8 @@ class RANSACRegressor(BaseEstimator, MetaEstimatorMixin, RegressorMixin):
     base_estimator : object, optional
         Base estimator object which implements the following methods:
 
-         * `fit(X, y)`: Fit model to given training data and target values.
-         * `score(X, y)`: Returns the mean accuracy on the given test data,
+         * fit(X, y) : Fit model to given training data and target values.
```

Comment:
Sorry I did not read the `git grep` output correctly. Leave this file as it was.